cpu/esp32/vendor/esp-idf/include/driver/rtc_io.h

```diff
@@ -0,0 +1,253 @@
+// Copyright 2015-2016 Espressif Systems (Shanghai) PTE LTD
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef DRIVER_RTC_GPIO_H
+#define DRIVER_RTC_GPIO_H
+
+#include <stdint.h>
+#include "esp_err.h"
+#include "driver/gpio.h"
+#include "soc/rtc_gpio_channel.h"
+#include "soc/rtc_periph.h"
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+typedef enum {
+    RTC_GPIO_MODE_INPUT_ONLY , /*!< Pad input */
+    RTC_GPIO_MODE_OUTPUT_ONLY, /*!< Pad output */
+    RTC_GPIO_MODE_INPUT_OUTPUT, /*!< Pad pull input + output */
+    RTC_GPIO_MODE_DISABLED,    /*!< Pad (output + input) disable */
+} rtc_gpio_mode_t;
+
+/**
+ * @brief Determine if the specified GPIO is a valid RTC GPIO.
+ *
+ * @param gpio_num GPIO number
+ * @return true if GPIO is valid for RTC GPIO use. talse otherwise.
```

Comment:
s/talse/false

Reply:
Ups, why didn't `codespell` catch it?

Reply:
Ah, this is vendor code and not checked by codespell. Therefore, I would leave as is..